pkg/controller/operators/operator_controller.go

```diff
@@ -119,6 +120,12 @@ func (r *OperatorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	in := &operatorsv1.Operator{}
 	if err := r.Get(ctx, req.NamespacedName, in); err != nil {
 		if apierrors.IsNotFound(err) {
+			// If the Operator instance is not found, we're likely reconciling because
```

Comment:
Nit: OLM could use a delete predicate here to filter delete events, but that's probably overkill. Non-blocking

Reply:
There _are_ cases where we want to see the delete events. If an `Operator` resource is deleted, but some of the components associated with that operator still exist, we should re-create the deleted `Operator` resource (IIUC).